docs/TuxStores.md

```diff
@@ -0,0 +1,172 @@
+          <ol style="list-style-type:lower-alpha">
+            <li><a href="#store-emitChange">METHOD: store.emitChange</a></li>
+            <li><a href="#store-addChangeListener">METHOD: store.addChangeListener</a></li>
+            <li><a href="#store-removeChangeListener">METHOD: store.removeChangeListener</a></li>
+          </ol>
+        </li>
+      </ol>
+      </li>
+    </ol>
+  </li>
+  <li><a href="#TuxStores-Complete-Example">TuxStores Complete Example</a>
+</ol>
+
+##<a id="Premise"></a>Premise [#](#Premise)
+>Tux Stores is a module providing the interface for extending Flux architecture stores with common methods.
+
+##<a id="Implementation"></a>Implementation [#](#Implementation)
+###<a id="Requiring-Stores"></a>Requiring Stores [#](#Requiring-Stores)
+Tux uses Facebook's [Flux](https://facebook.github.io/flux/) architecture and provides a convenience module that adds common methods to stores via a Stores interface. This interface is exposed via:
+
+```
+    var Stores = require('tux/Stores')
+```
+
+###<a id="Creating-Stores"></a>Creating Stores [#](#Creating-Stores)
+The `Stores` object allows the user to create stores via invoking [Stores.createStores](#createStores). Stores are a place in the Flux architecture to store and manipulate data. You can pass in the store methods that you have created to the `createStores` method and it will return an object that has been extended with the `emitChange`, `addChangeListener`, and `removeChangeListener` methods that are common to Flux stores. The Tux methods are added to the object being extended first, so if you want to overwrite Tux's implementation of one or all of those methods, you just need to provide your methods with those keys on the input object to `createStore`  Let's take a look at an example: [#](#create-store-example)
+<a id="create-store-example"></a>
+
+```
+    var messageStore = Stores.createStore({
+      _messages: {},
+      getAll: function () {
+        return Object.keys(this._messages);
+      }
+    });
+```
+
+Below is the API documentation for this method.
+
+###<a id="Stores-createStore"></a>Stores.createStore [#](#Stores-createStore)
+
+```
+    var store = Stores.createStore(methods);
+```
+
+####<a id="methods"></a>Parameter - `methods` - type: OBJECT - optional [#](#methods)
+Methods and properties to be extended on the store to be created. This object is optional and there are no expected keys.
+
+####<a id="storeReturn"></a>Return - `store` - type: OBJECT [#](#storeReturn)
+The `Stores.createStore` method returns an object which is described below.
+
+###<a id="store-emitChange"></a>Method `store.emitChange` - type: FUNCTION [#](#store-emitChange)
+A store instance is able to invoke the `emitChange` method, which will emit a change event string for any listeners that are listening for that particular string. The change event can be either the default event string `CHANGE` or one provided by the user.
```

Comment:
Remove the first sentence since.  Change ...for any listeners that are listening for that particular string... to (new sentence) Normally, it is common to invoke `emitChange` after data in the store is updated to alert the views to request the new data.